tests/test_colortable.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from prettytable import PrettyTable
+from prettytable.colortable import RESET_CODE, ColorTable, Theme
+
+
+@pytest.fixture
+def row_prettytable():
+    # Row by row...
+    row = PrettyTable()
+    row.field_names = ["City name", "Area", "Population", "Annual Rainfall"]
+    row.add_row(["Adelaide", 1295, 1158259, 600.5])
+    row.add_row(["Brisbane", 5905, 1857594, 1146.4])
+    row.add_row(["Darwin", 112, 120900, 1714.7])
+    row.add_row(["Hobart", 1357, 205556, 619.5])
+    row.add_row(["Sydney", 2058, 4336374, 1214.8])
+    row.add_row(["Melbourne", 1566, 3806092, 646.9])
+    row.add_row(["Perth", 5386, 1554769, 869.4])
+    return row
+
+
+@pytest.fixture
+def row_colortable():
+    row = ColorTable()
+    row.field_names = ["City name", "Area", "Population", "Annual Rainfall"]
+    row.add_row(["Adelaide", 1295, 1158259, 600.5])
+    row.add_row(["Brisbane", 5905, 1857594, 1146.4])
+    row.add_row(["Darwin", 112, 120900, 1714.7])
+    row.add_row(["Hobart", 1357, 205556, 619.5])
+    row.add_row(["Sydney", 2058, 4336374, 1214.8])
+    row.add_row(["Melbourne", 1566, 3806092, 646.9])
+    row.add_row(["Perth", 5386, 1554769, 869.4])
+    return row
+
+
+@pytest.fixture
+def color_theme():
+    return Theme(
+        default_color="31",
+        vertical_color="32",
+        horizontal_color="33",
+        junction_color="34",
+    )
+
+
+@pytest.fixture
+def char_theme():
+    return Theme(vertical_char=":", horizontal_char="~", junction_char="*")
```

Comment:
Codecov says this line isn't covered, so not used and can be removed?

Reply:
It can, let me do that.